[26.2] Failed release process for OperatorHub - community

Run: https://github.com/keycloak-rel/keycloak-rel/actions/runs/14403082781/job/40394932955

```
pull request create failed: GraphQL: Resource not accessible by integration (createPullRequest)
Error: Process completed with exit code 1.
```

The issue for the community matrix is invalid fork name inside of the `--head` attribute of creating a PR.

Current command:
```
gh pr create --title "operator keycloak-operator (26.2.0)" --repo https://github.com/k8s-operatorhub/community-operators.git --base main -F docs/pull_request_template.md --head k8s-operatorhub:26.2.0
```

The current `--head` param:
```
--head k8s-operatorhub:26.2.0
```

Correct `--head` param for keycloak-rel:
```
--head keycloak-rel:26.2.0
```

We didn't catch this in `keycloak-rel-testing` repository, as the `github.repository_owner` and the `env.repository_org` was the same.

It should be always the `github.repository_owner`.

I'll run a test in `keycloak-rel-testing` and then send a PR here. Not sure how we can avoid it.

--- a/.github/workflows/x-keycloak-operator-hub-publish.yml
+++ b/.github/workflows/x-keycloak-operator-hub-publish.yml
@@ -108,7 +108,7 @@ jobs:
           fi
           
           if [[ "$prOpen" == "false" ]]; then
-            gh pr create --title "operator keycloak-operator (${{ inputs.version }})" --repo ${{ env.repository-url }} --base main -F docs/pull_request_template.md --head ${{ env.repository-org }}:${{ inputs.version }} -r ${{ env.reviewers }}
+            gh pr create --title "operator keycloak-operator (${{ inputs.version }})" --repo ${{ env.repository-url }} --base main -F docs/pull_request_template.md --head ${{ github.repository_owner }}:${{ inputs.version }} -r ${{ env.reviewers }}
           else
             echo "PR already exists, skipping..."
           fi

